packages/perseus/src/multi-items/__tests__/__snapshots__/multi-renderer_test.js.snap

```diff
@@ -716,12 +716,8 @@ exports[`multi-item renderer should snapshot: initial render 1`] = `
                 <span>
                   <div>
                     <input
-                      aria-describedby="input-with-examples-aW5wdXQtbnVtYmVyIDE"
-                      autocapitalize="off"
-                      autocomplete="off"
-                      autocorrect="off"
```

Comment:
This also looks like it could be a regression. I see that the WB Text Input has at least an `autoComplete` prop.  I'm unsure how critical it will be that we lose the other two attributes, but I suspect it might be annoying on mobile to have something like autocapitalize and autocorrect on.